contracts/deposit-contract/depositContract.v.py

```diff
@@ -15,26 +15,17 @@
     index: bytes[8],
 })
 
-MIN_DEPOSIT_AMOUNT: public(uint256) # Gwei
```

Comment:
Can you add this back into the constructor? We still need it for our testnet deployments

Reply:
Thinking about this a bit more, @prestonvanloon what do you think ? 
This will hardcode a minimum deposit amount of 1eth into the contract.

We have been using 3.2 eth as our max deposit amount for our deployments. So
users would never have a failed transaction as long as they deposit the right amount.